router/router.go

```diff
@@ -104,7 +104,7 @@ func (r *router) logMemStats(interval time.Duration) {
 	for {
 		time.Sleep(interval)
 		runtime.ReadMemStats(&m)
-		r.log.Printf("MemStats: Alloc=%d Mallocs=%d Frees=%d NumGC=%d", m.Alloc, m.Mallocs, m.Frees, m.NumGC)
+		r.log.Printf("MemStats: Alloc=%d Mallocs=%d Frees=%d NumGC=%d HeapAlloc=%d", m.Alloc, m.Mallocs, m.Frees, m.NumGC, m.HeapAlloc)
```

Comment:
The documentation says that `Alloc` is the same as `HeapAlloc`: https://golang.org/pkg/runtime/#MemStats
Is this incorrect?
Also, I never saw these report different values (at least not with go1.15).